src/ashley/models.py

```diff
@@ -197,3 +205,46 @@ class Meta:
 
 # Default implementation of the AbstractLTIContext model
 LTIContext = model_factory(AbstractLTIContext)
+
+
+def image_directory_path(instance, filename):
+    """File will be uploaded to specific path"""
+    return "image_uploads/{0}/{1}/{2}".format(
+        instance.forum.id,
+        instance.poster.id,
+        f"{uuid.uuid4()}{pathlib.Path(filename).suffix}",
+    )
```

Comment:
You are mixing string interpolation styles (`.format` vs `f-strings`)
I suggest the following : 

```suggestion
    new_filename = "f"{uuid.uuid4()}{pathlib.Path(filename).suffix}"
    return f"image_uploads/{instance.forum.id}/{instance.poster.id}/{new_filename}"
```

Or maybe a longer, but maybe easier to understand: 

```suggestion
    target_directory = f"image_uploads/{instance.forum.id}/{instance.poster.id}"
    file_extension = pathlib.Path(filename).suffix
    new_filename = "f"{uuid.uuid4()}{file_extension}"
    return f"{target_directory}/{new_filename}"
```